carbon-p2-plugin/src/main/java/org/wso2/maven/p2/beans/CarbonArtifact.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
After completely refactoring of the feature plugin, it was released in 2015. These pull requests are related to that release. Also this is the 2nd pull request during the code migration from old repo to the new repo and the code you see is not clean. The refactored code can be found in later PRs (PR no 12.)